rocksdb.nim

```diff
@@ -90,34 +135,30 @@ template initRocksDB*(args: varargs[untyped]): Option[RocksDBInstance] =
   else:
     some(db)
 
-template getImpl(T: type) {.dirty.} =
-  if key.len <= 0:
-    return err("rocksdb: key cannot be empty on get")
-
-  var
-    errors: cstring
-    len: csize_t
-    data = rocksdb_get(db.db, db.readOptions,
-                       cast[cstring](unsafeAddr key[0]), csize_t(key.len),
-                       addr len, addr errors)
-  bailOnErrors()
-  if not data.isNil:
-    result = ok(toOpenArray(data, 0, int(len) - 1).to(T))
-    rocksdb_free(data)
-  else:
-    result = err("")
+proc get*(
+    db: RocksDBInstance,
+    key: openArray[byte],
+    onData: DataProc,
+    columnFamily = "default"): RocksDBResult[bool] =
 
-proc get*(db: RocksDBInstance, key: openArray[byte], onData: DataProc): RocksDBResult[bool] =
   if key.len <= 0:
     return err("rocksdb: key cannot be empty on get")
 
+  let columnFamilyHandle = db.validateColumnFamily(columnFamily)
+
   var
     errors: cstring
     len: csize_t
-    data = rocksdb_get(db.db, db.readOptions,
-                       cast[cstring](unsafeAddr key[0]), csize_t(key.len),
-                       addr len, addr errors)
+    data = rocksdb_get_cf(
+        db.db,
+        db.readOptions,
+        columnFamilyHandle,
+        cast[cstring](unsafeAddr key[0]),
```

Comment:
nim 2.x remove support for `unsafeAddr`, and only support `addr`. You need to templatize `unsafeAddr` for nim 2.x.

Reply:
Thanks, will do.